kolibri/plugins/facility/assets/src/views/users/sidePanels/AssignCoachesSidePanel.vue

```diff
@@ -264,6 +264,7 @@
       });
 
       function closeSidePanel() {
+        emit('clearSelection');
```

Comment:
This `closeSidePanel` is the general method we use to close the side panel, adding the clearSelection here is incorrect because it will always clear the selection when closing the side panel even if the operation was not completed. This should be moved to the `_handleAssign` method after closing the side panel.